bin/xcompile

```diff
@@ -45,7 +45,7 @@ do_cargo() {
         # Docker for Mac is unusably slow.
 
         if [[ ! -d "$root/target/sysroot" ]]; then
-            die "xcompile: fatal: run \`bin/xcompile bootstrap\` first"
+            die "$(uw "xcompile: fatal:") run \`bin/xcompile bootstrap\` from the repo root first"
```

Comment:
You shouldn't have to run this from the repo root. If you do, that's a bug. Can we fix that instead?

Reply:
Making the error message more visible seems good! Though the function name `uw` is not exactly descriptive.

Reply:
Mercifully easy fix: #4695.